### PR TITLE
Migrated Batch 8 (final) of Density

### PR DESCRIPTION
**Pull request by qur786**

### CODE REVIEW

**Review by maneetgoyal — APPROVED** [review]
👍

**Review by sumitnegi7 — APPROVED** [review]

**Comment by sumitnegi7** [comment]
@qur786  resolve merge conflicts before merging

File: src/data/density-constants.ts [hunk 1]
```diff
@@ -2385,6 +2385,471 @@ export const DIPPRDensityDictionary: Record<string, DIPPRDensityProps> = {
...
+    C1: 0.59595,
+    C2: 0.24314,
+    C3: 543.15,
+    C4: 0.24856,
+    minimumTemperature: 178.35,
+    maximumTemperature: 543.15,
+    densityAtMinimumTemperature: 8.8236,
+    densityAtMaximumTemperature: 2.4511,
+  },
+
```
Comment (saikat-samanta) [inline src/data/density-constants.ts:2807]: Water is missing. Ask @maneetgoyal sir, Before adding

Comment (qur786) [reply src/data/density-constants.ts:2807]: @sumitnegi7  water and o Terphenyl was added in the beginning of this file , so they need not be added anymore you can checkout it in the last of the file.

**Review by sumitnegi7 — APPROVED** [review]
LGTM :+1:

**Review by saikat-samanta — APPROVED** [review]

**Pull request merged**